typescript/utils/src/utils.ts

```diff
@@ -39,6 +45,62 @@ export function formatCallData<
   );
 }
 
+export const parseMultisigModuleMetadata = (
```

Comment:
I think it'd be nice to put some of these hardcoded numbers into consts, just like it's done in `MultisigModuleMetadata.sol`